network-api/networkapi/wagtailpages/templates/wagtailpages/research_breadcrumb.html

```diff
@@ -0,0 +1,32 @@
+{% comment %}
```

Comment:
**Suggestion** since this is a fragment and not a "page", I think it would best be in the "fragments" directory. 

I guess the page templates should possibly be moved into the "pages" directory too... but that is out of scope for this PR.

Reply:
Moved! Going to mark this as resolved